src/utils/trust/legal.entity.beneficial.owner.mapper.ts

```diff
@@ -55,6 +65,11 @@ const mapLegalEntityToSession = (
     };
   }
 
+  let stillInvolved: string | null = (formData.stillInvolved === "1") ? "Yes" : "No";
+
+  // If a boolean value isn't receieved from the web form (it could be null or undefined, e.g. if question not displayed), need to set null
+  if (!formData.stillInvolved) { stillInvolved = null; }
```

Comment:
`stillInvolved = null `should be on a separate line.

Reply:
or you can remove the curly bracket

if (!formData.stillInvolved) stillInvolved = null;

depending on your code style rules

Reply:
We usually add the braces.